### DISCUSSION TITLE
Re-Design of "PSST" application interface

### DISCUSSION THREAD

**Discussion by theharryn** [jpochyla/psst#155] [General]
hello good night, a friend recommended this project to me, and i was interested in being able to help with my knowledge of UX/UI design, to improve the interface and make it more pleasant and accessible, i count on your feedback :)

My portfólio: https://www.behance.net/HarrynLucas
My ID discord: Harryn#1368

**Comment by secondlnl** [comment]
Related issues: 
#71, #616, #476, #474, #335, (Looks like most discussed here) #203, #178 and #113
Related discussions:
#400
